Automattic/vip-go-mu-plugins
# Adds longer timeout to validate counts

## Description

Sometimes (often on bigger sites) our periodic validate count checks fail due to timeout. This change introduces a way to increase timeout and increases it to 5s (from 2s) for validating counts operation.

## Changelog Description


### Plugin Updated: Search

This change introduces a way to increase timeout and increases it to 5s (from 2s) for validating counts operation.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Add debug line: search/includes/classes/class-search.php:722 - `var_dump(['timeout', $timeout]);`
2. `dev-env exec -- wp vip-search health validate-posts-count` See the timeout being 5s
